docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -0,0 +1,185 @@
+| [`width`/`height`](#widthheight)        | Source image size                                                    | Change the size of the image.                                                                                                                                                                                                                                                                                                                                                                                 |
+| [`aspectRatio`](#aspectratio)           | Source image aspect ratio                                            | Force a specific ratio between the image's width and height.                                                                                                                                                                                                                                                                                                                                                  |
+| [`placeholder`](#placeholder)           | `"dominantColor"`/`DOMINANT_COLOR`                                   | Choose the style of temporary image shown while the full image loads.                                                                                                                                                                                                                                                                                                                                         |
+| [`formats`](#formats)                   | `["auto","webp"]`/`[AUTO,WEBP]`                                      | File formats of the images generated.                                                                                                                                                                                                                                                                                                                                                                         |
+| [`transformOptions`](#transformoptions) | `{fit: "cover", cropFocus: "attention"}`                             | Options to pass to sharp to control cropping and other image manipulations.                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Is there a reason you included these ones here again? What would you think about changing this section from "all options" to "additional sharp options" or something like that?

Reply:
I did it so there was a place to see them all, and make it clear that there's nothing special about the others, just that they're more common.

Reply:
Cool, that makes sense to me!